tests/modules/programs/rofi/custom-theme.rasi

```diff
@@ -15,3 +15,5 @@ border-color: #FFFFFF;
 foreground-color: rgba ( 250, 251, 252, 100 % );
 width: 512;
 }
+
+@import: "~/.cache/wal/colors-rofi-dark";
```

Comment:
Don't know if this being the last is an issue. AFAIK CSS/SASS are not position dependent, and since RASI is based on CSS it shouldn't be too.

Anyway, if this is position dependent probably the only solution would be to use `hm.types.dagOf`.

Reply:
> @import makes all variables, mixins, and functions globally accessible. This makes it very difficult for people (or tools) to tell where anything is defined.

Didn't read the entirety of the link, but I assume this means the ordering doesn't matter for declared variables and other stuff? That is, variable declaration can come after utilisation.

Reply:
Also, this isn't a valid CSS import rule. It should be `@import "stuff";`

Reply:
Yeah, true history.

Reply:
Fixed.